### ISSUE TITLE
Headings in Preview resource modal

### ISSUE THREAD

**Issue by astridu** [LD4P/sinopia_editor#4018]
The headings in Preview resource modal should be:
"Preview resource" = H1
Title of resource= H2

<img width="1050" height="291" alt="Image" src="https://github.com/user-attachments/assets/214215b9-1788-4642-9b28-84885ef72a32" />
